commands/snapshotcheckercommand.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2017 CN-Consult GmbH
+ * @author Jens Stahl <jens.stahl@cn-consult.eu>
+ *
+ * License: Please check the LICENSE file for more information.
+ */
+
+namespace PBST\Command;
+
+use ProfitBricksApi\Snapshot;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use ProfitBricksApi\ProfitBricksApi;
+use Exception;
+
+/**
+ * Class SnapshotCheckerCommand
+ *
+ * This command verifies the snapshots have been taken successful and sends an email to a configured address.
+ */
+class SnapshotCheckerCommand extends Command
+{
+    private $config;
+    private $virtualMachines = null;
+
+    protected function configure()
+    {
+        $this
+            ->setName("snapshot:checker")
+            ->setDescription("Checks, if all snapshots of a server have been done and send success notification.");
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if (is_readable('config.ini'))
+        {
+            $this->config = parse_ini_file('config.ini', true);
+            if (!isset($this->config['api']['user']) || !isset($this->config['api']['password'])) throw new Exception("No user or no password configured to connect ProfitBricks!");
+        }
+        else throw new Exception("Error during reading config.ini!");
+
+        $profitBricksApi = new ProfitBricksApi();
+        $profitBricksApi->setUserName($this->config["api"]["user"]);
+        $profitBricksApi->setPassword($this->config["api"]["password"]);
+        /** @var Snapshot[] $snapshots */
```

Comment:
This documentation is not wrong but it would better to document the called method ProfitBricksApi::snapshots() and define the return value by @return Snapshot[]. Then there would be no warnings from our IDE.